js/integration/jquery/events.js

```diff
@@ -9,12 +9,13 @@ if(useJQuery) {
         jQuery.event.special[name] = eventObject;
     });
 
-    var passiveListenerEventName = eventsEngine.passiveListenerEvents.nativeEventName;
-    jQuery.event.special[passiveListenerEventName] = {
-        setup: function(data, namespaces, handler) {
-            domAdapter.listen(this, passiveListenerEventName, handler, { passive: false });
-        }
-    };
+    eventsEngine.forcePassiveFalseEventNames.forEach(function(eventName) {
+        jQuery.event.special[eventName] = {
+            setup: function(data, namespaces, handler) {
+                domAdapter.listen(this, eventName, handler, { passive: false });
+            }
+        };
+    });
```

Comment:
Where tests on this functionality?

Reply:
There were simple test for this method, see [Fix Illegal Invocation errors...](https://github.com/IgnatovDan/DevExtreme/commit/42077826aafa30b216b6203acdbc387de19cfe04)
I added a new one for the complete event usage scenario: [Test how our code calls the add/removeEventListener methods with jQue…](https://github.com/DevExpress/DevExtreme/pull/7304/commits/afde85d4cc115036565f8c2017027409a60e92de)